### PR TITLE
Replace mqttVars vector with map

### PR DESCRIPTION
**Pull request by Alcoinus**
This isn't tested, as I use neither software BD nor scale. But as the changes are small, I believe there won't be any issues.

EDIT:
I continued to work on this and replaced the mqttVars vector with a map. This is intended to be a precursor to allow Home Assistant MQTT auto discovery.

This time, I have tested the changes and they work without issue for me ;)

### CODE REVIEW

File: src/debugSerial.cpp [hunk 1]
```diff
@@ -52,6 +52,19 @@ void debugPrintln(const char *message) {
     }
 }
 
+// Print to remote serial (e.g. using OTA Monitor Task ) if client is connected, otherwise use hardware serial
+void debugPrintln(const String& message) {
+    char time[12];
```
Comment (kjyv) [inline src/debugSerial.cpp:57]: better to not repeat basically the whole content of debugPrintln(const char *message) here, just call it:
debugPrintln(message.c_str());

Comment (Alcoinus) [reply src/debugSerial.cpp]: Thanks, I have updated it.

**Review by kjyv — APPROVED** [review]

File: src/main.cpp [hunk 2]
```diff
@@ -1818,6 +1808,46 @@ void setup() {
...
+    mqttVars["backflushON"] = []{ return &editableVars.at("BACKFLUSH_ON"); };
+    mqttVars["startUsePonM"] = []{ return &editableVars.at("START_USE_PONM"); };
+    mqttVars["startKp"] = []{ return &editableVars.at("START_KP"); };
+    mqttVars["startTn"] = []{ return &editableVars.at("START_TN"); };
+    mqttVars["aggKp"] = []{ return &editableVars.at("PID_KP"); };
+    mqttVars["aggTn"] = []{ return &editableVars.at("PID_TN"); };
+    mqttVars["aggTv"] = []{ return &editableVars.at("PID_TV"); };
+    mqttVars["aggIMax"] = []{ return &editableVars.at("PID_I_MAX"); };
+    mqttVars["steamKp"] = []{ return &editableVars.at("STEAM_KP"); };
+    if (ONLYPID == 0) {
```
Comment (fiendie) [inline src/main.cpp:1827]: Could use some blank lines here and there :)
We usually have one before an if-statement.

Comment (Alcoinus) [reply src/main.cpp:1828]: Thanks for the input. I have added some blank lines.

**Review by fiendie — DISMISSED** [review]

**Pull request merged**